### PR TITLE
Add default log type

### PR DESCRIPTION
**Pull request by kgromadzki**

### CODE REVIEW

File: whylog/config/abstract_config.py [hunk 1]
```diff
@@ -71,7 +80,7 @@ def _save_filename_matcher_definition(self, matcher_definition):
         pass
 
     def get_all_log_types(self):
-        return six.itervalues(self._log_types)
+        return itertools.chain(six.itervalues(self._log_types), self.DEFAULT_LOG_TYPE)
```
Comment (ppolewicz) [inline whylog/config/abstract_config.py]: Please invert the order

File: whylog/config/abstract_config.py [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
 from whylog.config.parser_name_generator import ParserNameGenerator
 from whylog.config.parser_subset import ConcatenatedRegexParser
 from whylog.config.rule import RegexRuleFactory
+from whylog.config.super_parser import RegexSuperParser
 
 
 @six.add_metaclass(ABCMeta)
 class AbstractConfig(object):
     words_count_in_name = 4
+    DEFAULT_LOG_TYPE = default_log_type = LogType(
```
Comment (ppolewicz) [inline whylog/config/abstract_config.py]: `DEFAULT_LOG_TYPE = LogType(`

File: whylog/config/abstract_config.py [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
+        "default", [
+            WildCardFilenameMatcher("", "", "default", RegexSuperParser("", [], {}))
```
Comment (ppolewicz) [inline whylog/config/abstract_config.py:22]: please test that it doesn't match anything

File: whylog/config/abstract_config.py [hunk 4]
```diff
@@ -1,18 +1,27 @@
...
+from whylog.config.log_type import LogType
 from whylog.config.parser_name_generator import ParserNameGenerator
 from whylog.config.parser_subset import ConcatenatedRegexParser
 from whylog.config.rule import RegexRuleFactory
+from whylog.config.super_parser import RegexSuperParser
 
 
 @six.add_metaclass(ABCMeta)
 class AbstractConfig(object):
     words_count_in_name = 4
```
Comment (ppolewicz) [inline whylog/config/abstract_config.py:19]: why is that constant lowercase?

Comment (kgromadzki) [inline whylog/config/abstract_config.py:19]: This variable can be override in subclasses

Comment (ppolewicz) [inline whylog/config/abstract_config.py:19]: ok, but still it should be uppercase (unless you want to make a getter, but here you don't)

**Pull request merged**